<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectTabHeader</class>
 <widget class="QWidget" name="EffectTabHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>35</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #444; color: white; border: 1px solid #555;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <widget class="QToolButton" name="startStopButton">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Start/Stop Effect</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #555; color: white;</string>
     </property>
     <property name="text">
      <string notr="true">▶</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="effectNameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string notr="true">Effect Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Remove effect</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #555; color: white;</string>
     </property>
     <property name="text">
      <string notr="true">✕</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

